toolkit/block-producer-metadata/rpc/src/lib.rs

```diff
@@ -1,3 +1,53 @@
+//! Crate providing Json RPC methods for the Block Producer Metadata feature of the Partner Chains Toolkit
+//!
+//! ## Contents
+//!
+//! This crate provides the [BlockProducerMetadataRpcApiServer] trait defining the JsonRPC methods related to
+//! block producer metadata and its concrete implementation [BlockProducerMetadataRpc]. Currently, only a
+//! single method is provided:
+//! - `pc_getBlockProducerMetadata`
+//!
+//! ## Usage - PC Builders
+//!
+//! To use the Json RPC service defined in this crate, first make your runtime implement
+//! [sp_block_producer_metadata::BlockProducerMetadataApi]. Eg. assuming the pallet `BlockProducerMetadataPallet`
+//! in your runtime uses `BlockProducerMetadataType` as the metadata type, the following should be included in your
+//! `impl_runtime_apis` block:
+//! ```rust, ignore
+//! impl BlockProducerMetadataApi<Block, BlockProducerMetadataType> for Runtime
+//! {
+//! 	fn get_metadata_for(
+//! 		cross_chain_pub_key: &CrossChainPublicKey,
+//! 	) -> Option<BlockProducerMetadataType> {
+//! 		BlockProducerMetadataPallet::get_metadata_for(&cross_chain_pub_key)
+//! 	}
+//! }
+//! ```
+//!
+//! Afterwards, the [BlockProducerMetadataRpc] Json RPC service can be added into your node's Json RPC stack, eg.:
```

Comment:
I don't want to spam suggestions about this over the whole PR but [I would use `of` for inanimate nouns instead of `'s`](https://blog.virtualwritingtutor.com/possessive-s-or-of-learn-which-to-use-and-why/). 
```suggestion
//! Afterwards, the [BlockProducerMetadataRpc] Json RPC service can be added into the Json RPC stack of your node, eg.:
```